README.md

```diff
@@ -263,6 +285,9 @@ OPTIONS
       [default: quay.io/eclipse/che-operator:nightly] Container image of the operator. This parameter is used only when 
       the installer is the operator
 
+  --debug
+      Enables the debug mode for Che server
```

Comment:
It could be useful to put here some info like: 

> after that, you'll be able to use `server:debug` command to connect to your Che Server via debug protocol
> But note that it's not recommended to do it on publicly available instances since there are a few modifications in Che Server configuration which does your Che non-production-ready.

I'm not sure about wording and messages. Just review it and think whether it really makes sense )